product_variant_configurator/models/product_configurator.py

```diff
@@ -147,15 +147,14 @@ def _onchange_product_attribute_ids_configurator(self):
         if not self.product_id:
             product_tmpl = self.product_tmpl_id
             values = self.product_attribute_ids.mapped("value_id")
-            if "partner_id" in self._fields:
+            if self._partner_id_field in self._fields:
```

Comment:
I don't think we should use these onchange helpers in v16, as `sale` is now using computed writable for everything. Better to overwrite that computed fields instead.

Reply:
Once again I don't understand what you want to do here.
This code is in `product_variant_configurator`, and it is used to re-define the `name` field (according to the partner language).
This method is used for `sale_variant_configurator` and `purchase_variant_configurator` to do this correctly.

Reply:
OK, don't worry. Let's continue this way.